runtime/near-vm-runner/tests/test_error_cases.rs

```diff
@@ -530,16 +525,16 @@ fn test_external_call_error() {
         );
     });
 }
-#[test]
```

Comment:
Why this change?

Reply:
This assert disk cache count from 0 to 1, which would fail due to on disk cache is not utilized